Reference/Templating/index.md

```diff
@@ -1,44 +1,26 @@
 ---
-versionFrom: 7.0.0
+versionFrom: 8.0.0
 ---
 
 # Templating
 
-_Templating in Umbraco consists of 2 larger concepts, namely Templates, and Macros. Templates are used for the HTML layout of your pages, whereas macros are reusable dynamic components used for embedding navigation, forms, lists, and so-on in your templates._
+_Templating in Umbraco consists of 3 larger concepts, namely Templates (Views), Partials (Partial Views) and Macros (Macro Partials). Templates are used for the HTML layout of your pages, partials can be included in your templates for shared functionality across different page templates and macros for reusable dynamic components that can be controlled by editors to embed functionality into the grid or rich text areas._
 
-## Templates
+## Templating technology
 
-There are 2 types of templating technologies in Umbraco:
+In Umbraco V8 there is now only one type of templating technology:
 
 * MVC (views)
-* WebForms (masterpages)
 
-By default Umbraco uses MVC templates.
+:::note
+ The WebForms (masterpages) and dynamic razor approaches to templating are still available in Umbraco 7, but are now no longer supported in V8.
+:::
 
-#### [Working with MVC (views, razor, etc...)](Mvc/index.md)
-
-Describes how to work with MVC views, the razor syntax and APIs available as well details on how to create forms, step-by-step guides and other advanced techniques.
-
-
-
-#### [Working with WebForms (masterpages, usercontrols, etc...)](Masterpages/index.md)
 
-Describes how to work with WebForms (masterpages) templating syntax and its various components like UserControls, etc...
 
-If you prefer to work with Webforms, you can change the default template engine in the */Config/umbracoSettings.config* file, find this section and set the type you'd like to use (*Mvc* or *WebForms*):
-
-```xml
-<templates>
-    <defaultRenderingEngine>WebForms</defaultRenderingEngine>
-</templates>
-```
-
-#### Hybrid templates
-
-Umbraco can work with both MVC and Webforms templates at the same time, however there can only be one default. Umbraco will first check for an MVC view file before it checks for a Webforms master page file. For example, if a template was created in the backoffice called "Home", then when rendering a content page with this template Umbraco will check these locations and use the first one that is found:
+#### [Working with MVC (views, razor, etc...)](Mvc/index.md)
 
-~/Views/Home.cshtml
-~/Masterpages/Home.master
+Describes how to work with MVC views, the razor syntax and APIs available as well details on how to create forms, step-by-step guides and other advanced techniques.
```

Comment:
```suggestion
Describes how to work with MVC views, the razor syntax and APIs available. It also describes how to create forms, has some step-by-step guides and other advanced techniques.
```
Found this sentence a bit confusing. Tried to make it less so, don't think it's great yet though 😁

Reply:
yes I didn't write that :-P